components/cam/src/physics/cam/shoc_intr.F90

```diff
@@ -317,6 +317,7 @@ subroutine shoc_init_e3sm(pbuf2d, dp1_in)
       call pbuf_set_field(pbuf2d, tk_idx, 0.0_r8) 
       call pbuf_set_field(pbuf2d, fice_idx, 0.0_r8)
       call pbuf_set_field(pbuf2d, tke_idx, tke_tol)
+      call pbuf_set_field(pbuf2d, alst_idx, 0.0_r8)
```

Comment:
Just a quick question about this step.  Does anyone "set" `aist`?

Reply:
Not that I know of, but it shouldn't need to be initialized since it is never used before it is diagnosed.  Similar to why alst was never initialized, until my length scale reformulation change from a few months ago put a kink in it.  If you think we should set "aist" here, i'm not opposed to it to prevent similar things from happening in the future, should someone need to use aist before it is diagnosed.

Reply:
Yeah, we might as well initialize aist to zero... because why not?

Also, what's preventing us from merging this PR once aist is initialized (since Chris Terai and perhaps others are waiting on it)? My understanding is that we expect AT to fail on SHOC_run_and_cmp so it won't ever automerge, but all other tests should pass (and in fact nothing else would test this change at all). It might be good to let AT confirm my expectations, but otherwise?

Reply:
I agree, it would be nice to have this merged before the next round of nightly tests go in -  I just want to see CDASH do something different :-) .

Dido on aist initialization.

Reply:
okay, I will fix this and merge after expected tests run (sorry I'm always confused about what tests are expected to pass/fail that I usually play it safe and just want for Automerge or for someone to tell me it's all cool and ready to merge).